README.md

```diff
@@ -179,12 +179,13 @@ web3 transaction ADDRESS_HASH
 ### Build a smart contract
 
 ```sh
-web3 contract build FILENAME.sol
+web3 contract build FILENAME.sol --compiler COMPILER_VERSION
```

Comment:
To me `compiler` sounds like you are specifying `solc` or an alternative. What about something like `compiler-version` or `solc-version`?